spaces/S000066/properties/P000204.md

```diff
@@ -0,0 +1,11 @@
+---
+space: S000066
+property: P000204
+value: false
+refs:
+    - mathse: 5012402
+      name: Answer to "Does the double origin plane have a cut point?"
+---
+
+The space remains {P38} after the removal of any finite
+number of points, as shown in {{mathse:5012402}}.
```

Comment:
That seem unnecessarily complicated.  Why not say the space remains connected after removing any single point?
Actually, it's a triviality.